packages/ssx-sdk/src/types.ts

```diff
@@ -30,6 +30,7 @@ export type SSXSession = {
     sessionKey: string;
     siwe: string;
     signature: string;
+    ens?: SSXEnsData;
```

Comment:
I think we can be much more opinionated here with what this provides. This is like enableDAOLogin, you should just be able to turn it on and it does what you think it does, but you can configure it further. I think a good interface for this is: (see updated `SSXEnsData` as well)
```
const ssxConfig = {
  // ... other config
  ens?: boolean | SSXEnsData
};


```

Reply:
Oh, this type is to describe the ENS data, not the configuration.